channels/sync_runner.go

```diff
@@ -123,6 +123,9 @@ func NewSyncRunner(funcSource string) (*SyncRunner, error) {
 		return nil, err
 	}
 
+	runner.SetConsoleLogCallback(func(s string) { base.Infof(base.KeySyncFuncLog, base.UD(s).Redact()) })
+	runner.SetConsoleErrorCallback(func(s string) { base.Errorf(base.KeySyncFuncLog, base.UD(s).Redact()) })
```

Comment:
I didn't think we'd need to Redact() here - won't that get handled by logTo inside Infof/Errorf?

Reply:
We'd need to pass in the input string to be formatted, i.e: `base.Infof(base.KeySync, "%s", base.UD(s))` to let the logging functions handle redaction, but I thought it was probably not worth doing the additional string formatting work.